src/status_im/ui/screens/profile/subs.cljs

```diff
@@ -4,6 +4,13 @@
             [status-im.utils.build :as build]
             [status-im.utils.platform :as platform]))
 
+(defn- node-version [{:keys [web3-node-version]}]
+  (str "status-go v" (or web3-node-version "N/A") ""))
```

Comment:
What is the purpose of the empty string suffix `""`?

Reply:
I have no clue, I just kept it from the original code :(